dataParsing/main.py

```diff
@@ -0,0 +1,17 @@
+instances = ['tai256c', 'tho150', 'wil50', 'sko100c', 'lipa80a', 'nug30', 'rou20', 'kra32', 'chr12c', 'bur26e']
+
+def read_file(file_name):
+    time = []
+    cost = []
+
+    f = open(file_name, "r")
+    f1 = f.readlines()
+    for x in f1:
+        time.append(int(x.split()[0]))

```

Comment:
split od razu splituje na spacji?
int od arraya castuje arraya na array intów?

Reply:
drugie pytanie olej bo widzę że `x.split()` to array a `x.split()[0]` już nie

Reply:
tak splituje